### DISCUSSION TITLE
Check for CA policy that blocks device code flow

### DISCUSSION THREAD

**Discussion by merill** [maester365/maester#444] [Ideas]
Device code flow should be blocked by default and only allowed for the specific identities that need it (ideally a specific location too).

**Comment by NZLostboy** [comment]
Good idea, I don't think this is supported in what-if yet is it?

Additionally do you think we should look only for this specific policy? or would you factor in hybrid joined and compliant device checks as well which can be used to "solve" the same issue?

I think for clarity looking for this specifically makes sense, just throwing it out there as an idea.

**Comment by merill** [comment]
This is now implemented with the CA policy. https://maester.dev/docs/tests/MT.1051/
